pydeconz/models/sensor/air_quality.py

```diff
@@ -78,6 +80,16 @@ def pm_2_5(self) -> int | None:
         """Air quality PM2.5 (µg/m³)."""
         return self.raw["state"].get("pm2_5")
 
+    @property
+    def air_quality_formaldehyd(self) -> int | None:
```

Comment:
Add supports_xxxxxx for the new properties as well

Reply:
done